examples/gnrc_lorawan/Makefile

```diff
@@ -0,0 +1,60 @@
+# name of your application
+APPLICATION = gnrc_lorawan
+
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += gnrc_netdev_default
+USEMODULE += auto_init_gnrc_netif
+USEMODULE += gnrc_lorawan
+USEMODULE += gnrc_pktdump
+
+BOARD ?= b-l072z-lrwan1
+RIOTBASE ?= ../../
+
+# Turn on developer helpers
+DEVELHELP ?= 1
+
+# use SX1276 by default
+DRIVER ?= sx1276
+
+USEMODULE += $(DRIVER)
+
+# Required for the cipher module */
+CFLAGS += -DCRYPTO_AES
+#
+# We can reduce the size of the packet buffer for LoRaWAN, since there's no IP
+# support. This will reduce RAM consumption.
+CFLAGS += -DGNRC_PKTBUF_SIZE=512
+
+
+########################### COMPILE TIME CONFIGURATION ########################
+# NOTE: The following options can be configured on runtime as well using
+# `ifconfig`
+
+# OTAA compile time configuration keys
+CFLAGS += -DLORAMAC_APP_KEY_DEFAULT=\{0x00\,0x00\,0x00\,0x00\,0x00\,0x00\,0x00\,0x00\,0x00\,0x00\,0x00\,0x00\,0x00\,0x00\,0x00\,0x00\}
```

Comment:
to me this escaped hex notation is not very user friendly, but that's not a blocker for this PR, just something we might want to improve on later.

Reply:
no problem, I can change it :)

Reply:
I just realized this is already defined in RIOT with that format. Can we leave it for later maybe? :)

Reply:
yes sure, as said this is non blocking - and rather an inconvenience to users.

Reply:
ok! thanks